pkg/actuators/machine/reconciler.go

```diff
@@ -251,6 +261,25 @@ func (r *Reconciler) update() error {
 	return r.requeueIfInstancePending(newestInstance)
 }
 
+func (r *Reconciler) prepareTags() ([]*ec2.Tag, error) {
```

Comment:
Can we use this in both the create and update cycle so that the tags are built up in the same way? Would be good to share as much between create and update as possible

Reply:
yes, already we are using buildTags() as common code, here additionally we are getting clusterID in this call.